src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -93,10 +85,12 @@ <h6 class="is-6 is-small-font">
         </ng-template>
       </ng-container>
       <ng-container *ngIf="from !== 'notification'">
-        <app-plugin-persist-data *ngIf="selectedGroup == 'developer' && pages.includes(from)"
+        <app-plugin-persist-data *ngIf="selectedGroup == 'Developer' && pages.includes(from)"
           [serviceStaus]="serviceStatus" [serviceName]="category?.name" [pluginName]="plugin">
         </app-plugin-persist-data>
       </ng-container>
+      <app-tab-navigation #tabNaigation *ngIf="groupTabs && !['notification', 'config-manager'].includes(from)"
```

Comment:
Are we good with Dispacther, Bucket/ gui-ml-models etc.?

Reply:
Until now we are not showing configuration of Dispacther, Bucket in GUI. But yes after Manage service integration we might need to add the check for those as well.